### DISCUSSION TITLE
升级到2.0.4版，添加的Alist链接出现错误

### DISCUSSION THREAD

**Discussion by jmheqiao** [AlistGo/alist#270] [Q&A] [answered]
大神，我把原来的2.0.3升级到2.0.4-fix版后，原先在2.0.3版添加的alist链接出现错误如下错误：
json: cannot unmarshal object into Go struct field PathResp.data of type []model.File
![image](https://user-images.githubusercontent.com/78342744/147865903-7eabe7bb-868f-4a79-ae7c-d07d5dc3da24.png)
新添加alist也出现同样错误。我用armbian和docker相同问题，不知是否我操作失误！

**Comment by xhofe** [comment]
修改了接口 忘记改这个了，等会改一下再发个fix。

**Comment by xhofe** [comment, marked as answer]
已经在v2.0.4-fix2中修复
